tests/stability/stressng.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o pipefail
+
+# General env
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../metrics/lib/common.bash"
+
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
+DOCKERFILE="${SCRIPT_PATH}/stressng_dockerfile/Dockerfile"
+IMAGE="docker.io/library/local-stressng:latest"
+CONTAINER_NAME="${CONTAINER_NAME:-stressng_test}"
+
+function main() {
+	local cmds=("docker")
+
+	init_env
+	check_cmds "${cmds[@]}"
+	check_ctr_images "${IMAGE}" "${DOCKERFILE}"
+	sudo -E ctr run -d --runtime "${CTR_RUNTIME}" "${IMAGE}" "${CONTAINER_NAME}" sh -c "${PAYLOAD_ARGS}"
+
+	# Run 1 iomix stressor (mix of I/O operations) for 20 seconds with verbose output
+	info "Running iomix stressor test"
+	IOMIX_CMD="stress-ng --iomix 1 -t 20 -v"
+	sudo -E ctr t exec --exec-id 1 "${CONTAINER_NAME}" sh -c "${IOMIX_CMD}"
```

Comment:
Instead of use a constant exec-id (i.e. `1`,`2`,..etc.) you could use a random name id, for example by generating a random name with the function `random_name` (lib/common.sh); this should help reducing the test fails in such cases where the 'exec-id' was previously used.